.github/workflows/experimental-wasm-release.yml

```diff
@@ -0,0 +1,48 @@
+name: WASM Experimental builds
+on:
+  push:
+    branches:
+      - experimental-wasm
+jobs:
+  build:
+    if: ${{ github.repository_owner == 'cloudflare' }}
+    name: Build and release `wrangler@wasm` to NPM
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Use Node.js 16.7
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.7
+          cache: "npm"
+
+      - uses: actions/cache@v2
+        id: node-modules-cache
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+
+      - name: Install NPM Dependencies
+        run: npm install
```

Comment:
```suggestion
        run: npm ci
```

Reply:
why don't we do this in our other workflows?

Reply:
We do it in the `pullrequest.yml` workflow. I forgot to update the `release.yml` workflow with the change.

Reply:
And I didn't spot it when it got into the `prerelease.yml` workflow. 😬